src/test/java/com/cloudant/tests/DesignDocumentsTest.java

```diff
@@ -52,38 +55,58 @@
 
     private static Database db;
     private CloudantClient account;
+    private File rootDesignDir;
+    private File designDocExample;
+    private DesignDocumentManager designManager;
 
 
     @Before
     public void setUp() {
         account = clientResource.get();
         db = dbResource.get();
+        rootDesignDir = new File(System.getProperty("user.dir")
+                + "/src/test/resources/design-files");
+        designDocExample = new File(String.format("%s/example_design_doc.js", rootDesignDir));
+        designManager = db.getDesignDocumentManager();
     }
 
     @Test
-    public void designDocSync() {
-        DesignDocument designDoc = db.design().getFromDesk("example");
-        db.design().synchronizeWithDb(designDoc);
+    public void designDocSync() throws MalformedURLException {
```

Comment:
Not sure this actually throws a `MalformedURLException` any more. As it is a test, why not just use `Exception`, the test will fail anyway if any exception propagates out.